### PR TITLE
Update references to py-rcsb_exdb_assets from development to master

### PR DESCRIPTION
**Pull request by brindakv**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -516,21 +516,21 @@ site_info_configuration:
...
   #
-  PDBX_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/development/dictionaries/mmcif_pdbx_v5_next.dic
-  RCSB_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/development/dictionaries/rcsb_mmcif_ext_v1.dic
-  IHMDEV_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/development/dictionaries/ihm-extension.dic
-  FLR_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/development/dictionaries/flr-extension.dic
-  VRPT_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/development/dictionaries/vrpt_mmcif_ext.dic
-  VRPT_DICT_MAPPING_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/development/dictionaries/vrpt_dictmap.json
+  PDBX_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/master/dictionary_files/reference/mmcif_pdbx_v5_next.dic
+  RCSB_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/master/dictionary_files/dist/rcsb_mmcif_ext.dic
+  IHMDEV_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/master/dictionary_file/reference/ihm-extension.dic
```
Comment (aliciaaevans) [inline README.md:521]:
```suggestion
  IHMDEV_DICT_LOCATOR: https://raw.githubusercontent.com/rcsb/py-rcsb_exdb_assets/master/dictionary_files/reference/ihm-extension.dic
```
typo

**Review by aliciaaevans — CHANGES_REQUESTED** [review]
Just one typo. Also `rcsb/db/config/exdb-config-example.yml` needs the update notes at the top of the file.

File: README.md (hunk 1 above)
Comment (aliciaaevans) [reply README.md:521]: Also, updating `HISTORY.txt` and the version in `__init__.py`. (unless that's not needed since no python code changed?)

**Comment by brindakv** [comment]
Thanks @aliciaaevans. I updated the header comment and fixed the typo. I am not sure if I should update `History.txt` and `__init__.py` here considering it is such a minor update.

**Review by aliciaaevans — APPROVED** [review]

**Comment by brindakv** [comment]
Based on our discussion today, this update involves the README and a config file only. So no version update is needed.

**Comment by piehld** [comment]
@brindakv Correct, and also need to update the mock-data.

**Comment by brindakv** [comment]
Thanks @piehld. Updated mock-data.

**Comment by brindakv** [comment]
I'll merge when the checks are completed.

**Comment by piehld** [comment]
Sounds good @brindakv, thanks for the updates!

**Pull request merged**